### DISCUSSION TITLE
Angular Hot module replacement and F5

### DISCUSSION THREAD

**Discussion by volkorros** [salesforce/akita#764] [Q&A]
Hello everyone! I need some help/
If using this approach https://github.com/datorama/akita/blob/master/docs/docs/angular/hmr.mdx and the developer clicks F5 (reload the page) all data are taken from the local storage. Is there any possibility to make HMR working and clear storage on page reload?
P. S. I also tried to add clearStorage in the store's constructor but it doesn't work each page reload.

**Comment by NetanelBasal** [comment]
Listen to beforeunload and clear the storage.
